systemtest/conftest.py

```diff
@@ -275,19 +315,19 @@ def dump_key(self, crash_id, name):
 
         return f"v1/{name}/{crash_id}"
 
-    def verify_stored_data(self, crash_id, raw_crash, dumps, s3conn):
+    def verify_stored_data(self, crash_id, raw_crash, dumps, storage_helper):
         # Verify the raw crash file made it
         key = self.raw_crash_key(crash_id)
-        assert key in s3conn.list_objects(prefix=key)
+        assert key in storage_helper.list_objects(prefix=key)
 
         # Verify the dump_names file made it
         key = self.dump_names_key(crash_id)
-        assert key in s3conn.list_objects(prefix=key)
+        assert key in storage_helper.list_objects(prefix=key)
 
         # Verify the dumps made it
         for name in dumps.keys():
             key = self.dump_key(crash_id, name)
```

Comment:
this is failing in CI after the change to not override name from "upload_file_minidump" to "dump" in gcs, and is fixed by the following change, which I am waiting to push so as to not interfere with willkg's review:
```diff
diff --git a/systemtest/conftest.py b/systemtest/conftest.py
index 0327f77..28beb21 100644
--- a/systemtest/conftest.py
+++ b/systemtest/conftest.py
@@ -75,6 +75,12 @@ class GcsHelper:
             "bucket": self.bucket,
         }

+    def dump_key(self, crash_id, name):
+        if name in (None, ""):
+            name = "dump"
+
+        return f"v1/{name}/{crash_id}"
+
     def list_objects(self, prefix):
         """Return list of keys in GCS bucket."""
         self.logger.info('listing "%s" for prefix "%s"', self.bucket, prefix)
@@ -120,6 +126,12 @@ class S3Helper:
         client = session.client(**client_kwargs)
         return client

+    def dump_key(self, crash_id, name):
+        if name in (None, "", "upload_file_minidump"):
+            name = "dump"
+
+        return f"v1/{name}/{crash_id}"
+
     def list_objects(self, prefix):
         """Return list of keys in S3 bucket."""
         self.logger.info('listing "%s" for prefix "%s"', self.bucket, prefix)
@@ -309,12 +321,6 @@ class CrashVerifier:
     def dump_names_key(self, crash_id):
         return f"v1/dump_names/{crash_id}"

-    def dump_key(self, crash_id, name):
-        if name in (None, "", "upload_file_minidump"):
-            name = "dump"
-
-        return f"v1/{name}/{crash_id}"
-
     def verify_stored_data(self, crash_id, raw_crash, dumps, storage_helper):
         # Verify the raw crash file made it
         key = self.raw_crash_key(crash_id)
@@ -326,7 +332,7 @@ class CrashVerifier:

         # Verify the dumps made it
         for name in dumps.keys():
-            key = self.dump_key(crash_id, name)
+            key = storage_helper.dump_key(crash_id, name)
             assert key in storage_helper.list_objects(prefix=key)

     def verify_published_data(self, crash_id, queue_helper):
```